src/shared/api/subsonic/subsonic-normalize.ts

```diff
@@ -146,6 +146,7 @@ const normalizeSong = (
         discNumber: item.discNumber || 1,
         discSubtitle: null,
         duration: item.duration ? item.duration * 1000 : 0,
+        explicitStatus: null,
```

Comment:
OpenSubsonic actually exposes this field. See https://opensubsonic.netlify.app/docs/responses/child/#:~:text=of%20the%20song.-,explicitstatus,-string